lua/treesitter-context.lua

```diff
@@ -1,6 +1,8 @@
 local api = vim.api
 
 local config = require('treesitter-context.config')
+local render = require('treesitter-context.render')
+local context = require('treesitter-context.context')
```

Comment:
These files were required on-demand in autocmd callbacks to improve loading time of the plugin.